src/qml/Messages.qml

```diff
@@ -706,7 +706,7 @@ Page {
 
             PropertyChanges {
                 target: pageHeader
-                title: " "
+                title: ""
```

Comment:
Plz remove empty properties, thats just unneeded code... also below.

Reply:
humm, i think it is usefull because it reset title property

Reply:
Ok in this case ^^